mi-api/src/state/postgres.rs

```diff
@@ -154,6 +155,11 @@ impl PgDb {
             .log_elapsed()
             .await
     }
+
+    #[instrument(skip(self), fields(elapsed) err, ret)]
+    pub async fn get_user_leaderboard(&self) -> Result<Vec<LeaderboardUser>, sqlx::Error> {
+        get_user_leaderboard(&self.pool).await
```

Comment:
would be nice to specify the namespace so it doesnt look like the function is calling itself

Reply:
Good point, going to open a PR for that.